package.json

```diff
@@ -57,9 +57,9 @@
     "test": "nyc --reporter=lcov --reporter=html --reporter=text-summary npm run test-bare",
     "lint": "eslint . --cache",
     "docs": "node scripts/generate-readme.js",
-    "integration-test-cleanup": "node scripts/integration-test-cleanup.js",
-    "simple-integration-test": "jest --maxWorkers=5 simple-suite",
-    "complex-integration-test": "jest --maxWorkers=5 integration",
+    "simple-integration-test": "jest --maxWorkers=5 integration-simple",
+    "thorough-integration-test": "jest --maxWorkers=5 integration-thorough",
+    "integration-test-cleanup": "node tests/utils/aws-cleanup.js",
```

Comment:
Maybe it'll be nicer to have scripts named as:

- `integration-test-run-basic`
- `integration-test-run-all`
- `integration-test-cleanup`

What do you think?

Then, if I see correctly, the _simple_ tests are about testing deployment of one lambda service and testing invocation of that lambda, while _thorough_ currently are about testing through API Gateway.

Maybe it'll be more future proof, to put what we have as _simple_ into:
`integration-tests/lambda` and with `integration-test-run-basic` just run this single _lambda_ test, while with 
`integration-test-run-all` run all (so _lambda_ and _apiGateway_) ?
Later we can do more exensive direct _lambda_ tests (if needed), and run some _basic_ test also against an API gateway

Reply:
I like that! Good suggestions. Will also update the packaging integration test to follow this naming convention.